stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe">    
+            </div>
+             
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+standard.storyName = 'Standard';
+standard.parameters = {
+    docs: {
+        storyDescription: 'The standard feed list item'
+    }
+};
+
+export const placeHolderImage = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item placeholder user image example">
+        <div class="fd-feed-list__body">
+            <div 
+                class="fd-avatar fd-avatar--s fd-avatar--placeholder sap-icon--person-placeholder fd-feed-list__thumb" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe">
+            </div>     
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+placeHolderImage.storyName = 'Placeholder User Image';
+placeHolderImage.parameters = {
+    docs: {
+        storyDescription: 'The Feed list item with placeholder user image'
+    }
+};
+
+export const withoutUserImage = () => `
+    <div class="fd-feed-list" aria-label="Feed List Item without user image example">
+        <div class="fd-feed-list__body">     
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+        </div>
+    </div>
+`;
+
+withoutUserImage.storyName = 'Without User Image';
+withoutUserImage.parameters = {
+    docs: {
+        storyDescription: 'The Feed list item without user image'

```

Comment:
The feed list item can be displayed without an image by removing the `fd-avatar` element from the container.